.github/CODEOWNERS

```diff
@@ -7,7 +7,7 @@
 
 # Catch all for non-code project files and unowned files | folders
 *                        @rickwinter @jhendrixMSFT
-/sdk/                    @rickwinter @chlowell @jhendrixMSFT @richardpark-msft @seankane-msft
+/sdk/                    @rickwinter @chlowell @richardpark-msft @seankane-msft
```

Comment:
This is a catch-all for packages that don't have owners.  Are there notifications that you see that you would like to filter out?  Perhaps we can pattern those out.

Reply:
Yes I don't want to see any PR notifications from all track 2 SDKs.  It's blowing up my PR folder so I can't track what actually needs my attention.

Reply:
Ok, this is fine for now.  I'll work on a solution to minimize what you see without turning everything off :)

Reply:
Filtering out `/sdk/**/arm*` would be good.  Is there a way to describe that?